CONTRIBUTING.rst

```diff
@@ -67,11 +67,10 @@ With respect to code standards, we follow `PEP8`_ and the `Google Style Guide`_.
 .. _`Google Style Guide` : https://google.github.io/styleguide/pyguide.html
 
 
-Licensing
+Licensing of contributions
 --------
 
-The License of this project is located in `LICENSE`_. By submitting a contribution to this project, you are agreeing that your contributions under this license and
-with this waiver of copyright interest.
+By submitting a contribution to this project, you are granting the project maintainers the ability to release your contribution under the project license located in `LICENSE`_. Along with this, you also waiver copyright interest.
```

Comment:
(I actually went through the licenses and TOS (for rigor's sake) had wanted to convey this far earlier; there is section 5 of Apache 2.0 for submission of contributions as @dmasad stated, there is also section D.6 in Github TOS which was only recently added [this year](https://github.com/blog/2327-new-github-terms-of-service-are-in-effect) to ease this for a sane default / to reduce bureaucracy excess. And should the Github servers experience a downtime, `git` is always there, which is distributed.)